controllers/syncer/csi_node.go

```diff
@@ -123,12 +127,12 @@ func (s *csiNodeSyncer) ensureContainersSpec() []corev1.Container {
 	})
 
 	nodePlugin.SecurityContext = &corev1.SecurityContext{
-		Privileged:               util.True(),
-		AllowPrivilegeEscalation: util.True(),
+		RunAsNonRoot: util.False(),
+		Privileged:   util.True(),
+		RunAsUser:    func(uid int64) *int64 { return &uid }(0),
 	}
```

Comment:
Node plugin need above privileges for mounting. It must run as root user as well.